### DISCUSSION TITLE
How to rectify the error - "Cannot read properties of undefined (reading 'user')"

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3217] [Q&A]
Hi,

When I'm running the `vercel dev` command, it is showing an error, can anyone please explain how to solve this? 

Earlier there was no such error, but now I don't know why this error is showing up, like in the `try catch` section in the `api` folder, it is directly going into the catch if I'm trying to fetch something from the fetcher file. This is just a sample.

But when I'm trying to run this, it is working
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/58f1cdf3-1898-40a7-850e-1417dfa33ccf)
<br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/77444403-cf5c-4539-831a-be887206c209)

<br><br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/8fb1f226-0c31-4991-b23b-6827691604d7)
<br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/81c080a4-5636-4760-add6-64198b7b454a)

**Comment by rickstaa** [comment]
@ yaten2302 It's hard to say what goes wrong in your code without having the changes you made. Can you maybe send me a git patch file of your current working tree (see https://www.specbee.com/blogs/how-create-and-apply-patch-git-diff-and-git-apply-commands-your-drupal-website)?

**Reply by yaten2302** [reply]
Hi @rickstaa, thank you so much for your help. Actually there was some problem with my code, it's rectified now, so I closed the discussion.
